src/DeferredFutures.jl

```diff
@@ -53,6 +54,8 @@ end
 Serialize a DeferredFuture such that it can de deserialized by `deserialize` in a cluster.
 """
 function Base.serialize(s::AbstractSerializer, ref::DeferredFuture)
+    Base.Serializer.serialize_cycle(s, ref) && return
```

Comment:
Finally figured out how to recreate the error that manifests itself in Dispatcher.jl. Tests have been updated.